docs/source/fabric_definition.rst

```diff
@@ -279,6 +279,17 @@ specifying:
   In summary, the example shows how a termination tile can be used to provide more complex interface blocks and all this can be easily modelled and implemented with FABulous.
   
   .. note::  The ``destination_name`` is refering to the port name used at the destination tile. FABulous will throw an error if the destination tile does not provide that port name.
+
+  Aside from ``BEGIN`` and ``END``, there also exist ``MID`` ports, which are used for wires spanning over two tiles.
```

Comment:
This phrasing might be slightly less ambiguous?

```suggestion
  Aside from ``BEGIN`` and ``END``, there also exist ``MID`` ports, which can be used for wires spanning more than two tiles.
```

Reply:
Yeah seems better! Thanks!